index.html

```
@@ -495,105 +552,94 @@ <h2 style="color: #fff; margin-bottom: 20px">Ready to Get Started?</h2>
+            aria-live="polite"
+            style="position: absolute; left: -9999px"
+          ></span>
+          <code>
+            &lt;link rel="stylesheet"
+            href="https://cdn.jsdelivr.net/gh/AOSSIE-Org/SocialShareButton@v1.0.4/src/social-share-button.css"&gt;
+            &lt;script
+            src="https://cdn.jsdelivr.net/gh/AOSSIE-Org/SocialShareButton@v1.0.4/src/social-share-button.js"&gt;&lt;/script&gt;
+          </code>
+          <code>
+            import { component$ } from '@builder.io/qwik'; import { SocialShareButton } from
+            './social-share-button-qwik'; export default component$(() => ( &lt;SocialShareButton
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)